dotnet/src/webdriver/DriverService.cs

```diff
@@ -219,20 +220,23 @@ protected virtual bool IsInitialized
                 bool isInitialized = false;
                 try
                 {
-                    Uri serviceHealthUri = new Uri(this.ServiceUrl, new Uri(DriverCommand.Status, UriKind.Relative));
-                    HttpWebRequest request = HttpWebRequest.Create(serviceHealthUri) as HttpWebRequest;
-                    request.KeepAlive = false;
-                    request.Timeout = 5000;
-                    HttpWebResponse response = request.GetResponse() as HttpWebResponse;
-
-                    // Checking the response from the 'status' end point. Note that we are simply checking
-                    // that the HTTP status returned is a 200 status, and that the resposne has the correct
-                    // Content-Type header. A more sophisticated check would parse the JSON response and
-                    // validate its values. At the moment we do not do this more sophisticated check.
-                    isInitialized = response.StatusCode == HttpStatusCode.OK && response.ContentType.StartsWith("application/json", StringComparison.OrdinalIgnoreCase);
-                    response.Close();
+                    using (var httpClient = new HttpClient())
```

Comment:
Probably would be better to keep this instance as private field in class.

Reply:
do you mean so it can be reused to stop the service?

Reply:
Let's keep it simple for now, we don't expect a lot of http requests to reuse client.